Jenkinsfile

```diff
@@ -0,0 +1,27 @@
+// vim: ft=groovy
+
+properties([
+    parameters([
+        string(name: 'TARGET', defaultValue: '', description: '')
+    ])
+])
+
+node('arm64') {
+    deleteDir()
+    checkout scm
+    stage('Build a Factory') {
+        sh 'git clone https://github.com/liquidinvestigations/factory'
+    }
+    stage('Prepare cloud image') {
+        sh 'factory/factory prepare-cloud-image'
```

Comment:
This should pull an image from somewhere, not build it.

Reply:
Yes, good catch.